YatimaStdLib/RBMap.lean

```diff
@@ -8,28 +8,21 @@ def single (a : α) (b : β) : RBMap α β cmp :=
   RBMap.empty.insert a b
 
 def enumList (xs : List α) : RBMap α Nat cmp :=
-  RBMap.ofList $ xs.enum.map (fun (x, y) => (y, x))
+  RBMap.ofList (xs.enum.map (fun (x, y) => (y, x))) cmp
 
 def unitMap (xs : List α) : RBMap α Unit cmp :=
-  RBMap.ofList $ xs.map (fun x => (x, ()))
+  RBMap.ofList (xs.map (fun x => (x, ()))) cmp
 
-def filterOut [BEq α] [Ord α] (map : RBMap α β cmp) (t : RBTree α cmp) :
+def filterOut [BEq α] [Ord α] (map : RBMap α β cmp) (t : Lean.RBTree α cmp) :
```

Comment:
Isn't there an `RBTree` on `Std`?

Reply:
Oh but it looks like it's just `RBSet` now: https://github.com/leanprover/std4/commit/9f3199c24f811b0e9fbeeafbb0ad2db224fe8e58
I'll fix this